CRVResponse/fcl/prolog.fcl

```diff
@@ -1,2 +1 @@
-#include "Offline/CRVReco/fcl/prolog_v11.fcl"
```

Comment:
Of course it's correct to remove the reference to CRVReco from here.
We have to make sure that all Production fcl files that do reco things start pointing to the correct prolog.

Reply:
I have tested the relevant production scripts (with the included tag). I don't know about any private scripts or scripts in other repos, can you take care of migrating those (if they exist)?

Reply:
I'm not aware of any other fcl files outside the standard Production scripts. If they work, then everything is Ok.